manifests/ulimit.pp

```diff
@@ -27,31 +27,14 @@
     }
   }
 
-  file { "/etc/systemd/system/${redis::service_name}.service.d/":
-    ensure                  => 'directory',
-    owner                   => 'root',
-    group                   => 'root',
-    selinux_ignore_defaults => true,
+  systemd::service_limits { "${redis::service_name}.service":
```

Comment:
What do you think about moving it into instance.pp? https://github.com/voxpupuli/puppet-redis/commit/2c2a3223aa1338985cc056355fb9f98fe0acb684